helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,80 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "amazon-cloudwatch-observability.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Name for cloudwatch-agent
+*/}}
+{{- define "cloudwatch-agent.name" -}}
+{{- default "cloudwatch-agent" .Values.agent.name }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "amazon-cloudwatch-observability.labels" -}}
+{{ include "amazon-cloudwatch-observability.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: EKS
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "amazon-cloudwatch-observability.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "amazon-cloudwatch-observability.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "amazon-cloudwatch-observability.managerServiceAccountName" -}}
+{{- if .Values.manager.serviceAccount.create }}
+{{- default (printf "%s-controller-manager" (include "amazon-cloudwatch-observability.name" .)) .Values.manager.serviceAccount.name }}
+{{- else }}
+{{- default "default" .Values.manager.serviceAccount.name }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "cloudwatch-agent.serviceAccountName" -}}
+{{- if .Values.agent.enabled }}
+{{- default (include "cloudwatch-agent.name" .) .Values.agent.serviceAccount.name }}
```

Comment:
The service account name expected is "amazon-cloudwatch-agent-operator-agent" does this create that?

Reply:
We are updating the naming of the components for the ones associated with CWA to remain consistent with historical naming. So this is an intentional change to not call it that anymore.